oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileStoreUtils.java

```diff
@@ -75,6 +77,11 @@ public static BufferedOutputStream createOutputStream(File file, Compression alg
         return new BufferedOutputStream(algorithm.getOutputStream(out));
     }
 
+    public static BufferedOutputStream createOutputStream(Path file, Compression algorithm) throws IOException {
+        OutputStream out = Files.newOutputStream(file);
+        return new BufferedOutputStream(algorithm.getOutputStream(out));
+    }
+
```

Comment:
can we remove this? It is never used and the class is deprecated.

Reply:
Done.